integration/spark-common/src/main/scala/org/apache/carbondata/spark/util/GlobalDictionaryUtil.scala

```diff
@@ -867,6 +868,11 @@ object GlobalDictionaryUtil {
         columnSchema,
         false
       )
+      if (!FileUtils
+        .validateTableExists(carbonTablePath.getMetadataDirectoryPath  + CarbonCommonConstants.FILE_SEPARATOR +
+                             tableIdentifier.getTableId)) {
+        throw new Exception(s"Table does not exists: ${ tableIdentifier.getTableUniqueName }")
+      }
```

Comment:
Remove this check from executor and follow the same method to handle as mentioned in above comment